kythe/cxx/indexer/textproto/analyzer.cc

```diff
@@ -523,7 +523,7 @@ Status AnalyzeCompilationUnit(const proto::CompilationUnit& unit,
 
   Status status = analyzer.AnalyzeSchemaComments(*file_vname, *descriptor);
   if (!status.ok()) {
-    return status;
+    LOG(ERROR) << "Error analyzing schema comments: " << status;
```

Comment:
Can we emit a diagnostic? At least in theory those might show up in the UI at some point.

Reply:
sgtm. Is there a canonical/preferred way to construct the vname for the diagnostic? I see in `KytheEntrySets.java`, there's some code to generate a signature based on the hash of the contents. I could do something similar here, but right now I'm just using "schema_comments" as the signature since there will be at most one of them.

Reply:
That made sense to me; if there's a future need to change the signature it's pretty simple to hash the contents or something as you say.

Reply:
:+1: